test/functions/bradc/resolution/intChoices.good

```diff
@@ -8,8 +8,8 @@ int8or64 variants:
 64
 64
 8
-64
-64
+8
+8
```

Comment:
Assuming this is another case of "We now go to the smaller size because its formal type can dispatch to the larger", over the weekend, I was wondering what would happen if we just eliminated this part of the rule (or eliminated it for params).

Reply:
I would expect, if we eliminated that part of the rule, we would get ambiguity in this case (and more generally if you haven't provided all of the int width overloads). Are you hoping for ambiguity in this case?

Reply:
I've adjusted things so that `intChoices` works as it did on main. To do so, I added a new rule in the pairwise formal comparison

> If one of the formals requires a param narrowing conversion and the other is not, the one not requiring such narrowing is better

Initial testing looks like this will work OK but I will run more testing.

Reply:
I ran more testing & there were just 2 failures in tests of disambiguation / coercion

Reply:
Sounds great!  (and definitely removes part of the diff that I was not looking forward to reviewing... glad I dragged my feet!)

With respect to your question above, I would've been open to ambiguity in this case (though I think I prefer the current behavior).  For non-params, I would not want ambiguity, though...  which suggests that I probably would prefer not having ambiguity for this case as well, since it continues the theme of making param cases behave more like their non-param equivalents in terms of dispatch.

Reply:
Oh, also:  What were the 2 (new?) failures?

Reply:
The 2 new failures are

```
[Error matching program output for functions/bradc/resolveIntLiterals]
[Error matching program output for functions/ferguson/coercions]
```

These are basically the same pattern. I have updated their .good files. Also this change allowed me to remove a behavior difference from the PR message (since it was just the case from resolveIntLiterals).